skeletonkey/config.py

```diff
@@ -100,6 +98,7 @@ def __getattr__(self, name):
             raise AttributeError(message + f" Please specify '{name}' in your config yaml.")
     
     def instantiate(self, **kwargs):
+        from .instantiate import instantiate

```

Comment:
Why did you do this? Shouldn't this be defined at the top of this file?

Reply:
This prevents circular imports by waiting for instantiate to be fully resolved before importing it.